__init__.py

```
@@ -38,46 +45,51 @@
-            params = {
-                "username": self.settings["geonames_user"],
-                "lat": lat,
-                "lng": lon
-            }
-            ocean_names = "http://api.geonames.org/oceanJSON"
-            land_names = "http://api.geonames.org/countryCodeJSON"
-
-            # reverse geo
-            data = requests.get(ocean_names, params=params).json()
+            toponym = "The " + data['ocean']['name']
+        except:
```

Ruff

Do not use bare `except`

(E722)